index.html

```diff
@@ -53,7 +53,11 @@
 				</div>
 				<div class="col-md-8">
 					<h2>Examples</h2>
-					<style shim-shadowdom>
+					<style is="custom-style">
+						paper-tabs /deep/ paper-ripple {
```

Comment:
use the `--paper-tab-ink` var instead of `/deep/` 

https://elements.polymer-project.org/elements/paper-tabs?active=paper-tab